server/ServerApp.cpp

```diff
@@ -963,7 +963,7 @@ void ServerApp::LoadChatHistory() {
         m_python_server.SetCurrentDir(GetPythonChatDir());
         // Call the Python load_history function
         success = m_python_server.LoadChatHistory(m_chat_history);
-    } catch (boost::python::error_already_set err) {
+    } catch (const boost::python::error_already_set &err) {
```

Comment:
prefer `type_name& var_name` to `type_name &var_name`

Reply:
Done